app/controllers/tasks_controller.rb

```diff
@@ -1,53 +1,68 @@
 class TasksController < ApplicationController
-  load_and_authorize_resource :project
-  load_and_authorize_resource :task, through: :project
-
+  load_and_authorize_resource :only => [:my_tasks, :update_task_logtime]
+  load_and_authorize_resource :project, except: [:my_tasks, :update_task_logtime]
+  load_and_authorize_resource :task, through: :project, except: [:my_tasks, :update_task_logtime]
+ 
   def create
     @task.set_status
     if @task.save
-      redirect_to project_tasks_url(@task.project), notice: "Returing from the create"
+      redirect_to project_tasks_url(@task.project), notice: 'Returing from the create'
     else
-      render new_project_task_path(@task.project), notice: "Cannnot enter data due to constraints"
+      render new_project_task_path(@task.project), notice: 'Cannnot enter data due to constraints'
     end
   end
 
   def update
     if !params[:assignable_employee_id].empty? && !params[:assignable_team_id].empty?
-      render :edit, notice: "Make sure you delete employee field or team field"
+      render :edit, notice: 'Make sure you delete employee field or team field'
     else
-      params[:task][:status] = "new" if params[:task][:assignable_id].empty?
-      params[:task][:status] = "assigned" if !params[:task][:assignable_id].empty?
+      params[:task][:status] = 'new' if params[:task][:assignable_id].empty?
```

Comment:
`Task::NEW_STATUS`

Reply:
`NEW_STATUS = 'new'.freeze
ASSIGNED_STATUS = 'assigned'.freeze
`

write this is task model
and use 
`
Task::ASSINGNED_STATUS
Task::NEW_STATUS
`